jsdrupal/drupal-admin-ui
# issue-474: Added input label for actions and fixed console errors.

#474 

## Screenshot / UI changes

![screen shot 2018-09-14 at 1 53 36 pm](https://user-images.githubusercontent.com/15143023/45538800-a79a2d00-b825-11e8-8abf-36d122defe34.png)
![screen shot 2018-09-14 at 1 54 44 pm](https://user-images.githubusercontent.com/15143023/45538845-c698bf00-b825-11e8-9494-936f9853afa2.png)

## Testing instructions

Go to taxonomy and taxonomy term overview page and check for the actions input label
Go to taxonomy term overview page and check the console for errors.
